src/get_y_eq.jl

```diff
@@ -0,0 +1,18 @@
+function get_y_eq(x_eqs::Vector{Vector{Nemo.AbstractAlgebra.RingElem}}, y_eqs::Vector{Vector{Nemo.AbstractAlgebra.RingElem}}, n::Int, m::Int, s::Int, u::Int, gens_Rjet)
```

Comment:
This function shares a lot of code with `get_x_eq`, I think it would be great have a single more general function. Maybe actually for prolonging a single equation.